.vscode/settings.json

```diff
@@ -5,8 +5,7 @@
     "**/.hg": true,
     "**/CVS": true,
     "**/.DS_Store": true,
-    "**/node_modules": true,
-    "**/dist/**": true
```

Comment:
This just hides it in the vscode file tree right?

Reply:
Yep.